tests/pytorch_tests/model_tests/feature_models/bn_folding_test.py

```diff
@@ -132,4 +131,18 @@ def compare(self, quantized_models, float_model, input_x=None, quantization_info
             is_bn_in_model = torch.nn.BatchNorm2d in [type(module) for name, module in quant_model.named_modules()]
 
         self.unit_test.assertTrue(self.fold_applied is not is_bn_in_model)
-        self.unit_test.assertTrue(np.isclose(out_quant, out_float, atol=1e-6, rtol=1e-4).all())
+
+        # Checking on multiple inputs to reduce probability for numeric error that will randomly fail the test
+        input_x2 = self.generate_inputs(self.create_inputs_shape())
+
+        out_float1 = torch_tensor_to_numpy(float_model(*input_x))
```

Comment:
instead of duplicating the code, use larger batch size for input_x and check each batch separately.